python/cuml/cuml/tests/test_device_selection.py

```diff
@@ -143,6 +146,23 @@ def make_reg_dataset():
     )
 
 
+def make_class_dataset(n_classes):
```

Comment:
How about the following as a name? Seems a bit more descriptive than using `class` which at first made me think of classes in terms of programming, not classes in classification
```suggestion
def make_multiclass_dataset(n_classes):
```